ci/concourse/pipelines/kf-pipeline.yml

```diff
@@ -261,3 +269,58 @@ jobs:
       run:
         dir: kf-master
         path: hack/upload-release.sh
+
+# Cut a new release of kf
+# TODO: Consider semantic versioning resource:
+# https://github.com/concourse/semver-resource#gcs-driver
+# ATM, it seems like overkill and we should consider it when we are
+# releasing in a more automated way.
+- name: kf-release
+  plan:
+  - aggregate:
+    - get: kf-master
+      trigger: false
+  - task: create-release
+    config:
+      platform: linux
+      image_resource:
+        type: docker-image
+        source: *ci-image-source
+      params:
+        SERVICE_ACCOUNT_JSON: *release_service_account_json
+        GCP_PROJECT_ID: *gcp_project_id
+        KO_DOCKER_REPO: *ko_docker_release_repo
+        RELEASE_BUCKET: *release_bucket
+        CLI_RELEASE_BUCKET: *cli_release_bucket
+        VERSION_MAJOR: 0
+        VERSION_MINOR: 100
```

Comment:
Will these ever change?

Reply:
I figured we would set it manually for now and update it as needed. The minor should be 1 ATM. WDYT?

Reply:
Setting it manually answers my main questions and sounds like a good approach. Should we make the minor a ((parameter)) so it's not possible to apply the pipeline unless the param is provided?

Reply:
Meaning the value is stored in credhub?

Reply:
I was imagining `fly set-pipeline -v VERSION_MINOR=0.1 ...`

Reply:
(assuming by "set it manually" you meant setting it in the pipeline yaml and re-applying)

Reply:
Ah I see. I was actually talking about being even less fancy. Like just committing the value in the pipeline.

Reply:
per our offline conversation: I moved the versions to parameters so they can be set via credhub and we won't have to worry about setting them in normal circumstances.